pkpdapp/pkpdapp/api/views/efficacy_experiment.py

```diff
@@ -9,13 +9,13 @@
     NotADatasetDose,
     CheckAccessToProject,
 )
-from pkpdapp.api.serializers import EfficacySerializer
+from pkpdapp.api.serializers import EfficacyExperimentSerializer
 from pkpdapp.models import EfficacyExperiment
 
 
-class DoseView(viewsets.ModelViewSet):
+class EfficacyExperimentView(viewsets.ModelViewSet):
     queryset = EfficacyExperiment.objects.all()
-    serializer_class = EfficacySerializer
+    serializer_class = EfficacyExperimentSerializer
     permission_classes = [
         IsAuthenticated & NotADatasetDose & CheckAccessToProject
```

Comment:
`NotADatasetDose` will error here. It's been accidentally copied over from the `Dose` view.